desktop/models/sale.coffee

```diff
@@ -39,6 +39,8 @@ module.exports = class Sale extends Backbone.Model
 
   bidUrl: (artwork) ->
     "#{@href()}/bid/#{artwork.id}"
+  
+  predictionUrl: -> "#{process.env.PREDICTION_URL}/#{@get('slug')}"
```

Comment:
Inside of Force I don't think its safe to rely on `process.env` due to its isomorphism. For environmental variables, use `sharify` as seen here: https://github.com/artsy/force/blob/master/desktop/apps/artist/client/index.coffee#L1